src/components/Tooltip/index.tsx

```diff
@@ -103,13 +119,22 @@ export function MouseoverTooltipContent({
   ...rest
 }: Omit<TooltipContentProps, 'show'>) {
   const [show, setShow] = useState(false)
-  const open = useCallback(() => {
+  const open = () => {
     setShow(true)
     openCallback?.()
-  }, [openCallback])
-  const close = useCallback(() => setShow(false), [setShow])
+  }
+  const close = () => {
+    setShow(false)
+  }
+
   return (
-    <TooltipContent {...rest} show={!disableHover && show} content={disableHover ? null : content}>
+    <TooltipContent
+      {...rest}
+      open={open}
+      close={close}
```

Comment:
you can technically just define this inline to be cleaner and get rid of lines 126-129

```
close={() => setShow(false)}
```

Reply:
close is used again on line 141 so i feel like defining the function actually makes sense here. also same thing for open, so want to keep it consistent for readability.